### ISSUE TITLE
balanceOf error 

### ISSUE THREAD

**Issue by femtoeu** [lmittmann/w3#27]
#16 same issue.
Trying to get balance of BUSD on Binance Smart Chain

```
        client := w3.MustDial("https://rpc.ankr.com/bsc")
	err = client.Call(
		eth.CallFunc(funcBalanceOf, contractAddress, address).Returns(&tokenBalance),
	)
	if err != nil {
		log.Fatal(err)
	}
```

It returns
```
w3: response handling failed: invalid argument 0: json: cannot unmarshal non-string into Go struct field CallArgs.gas of type hexutil.Uint64
```

**Comment by lmittmann** [comment]
I can not reproduce the issue. Could you provide a full code example? Which version are you on?

**Comment by femtoeu** [comment]
> I can not reproduce the issue. Could you provide a full code example? Which version are you on?

```
package main

import (
	"log"
	"math/big"

	"github.com/ethereum/go-ethereum/common"
	"github.com/ethereum/go-ethereum/core/types"
	"github.com/lmittmann/w3"
	"github.com/lmittmann/w3/module/eth"
)

func main() {
	client := w3.MustDial("https://rpc.ankr.com/bsc")

	contractAddress := w3.A("0xe9e7CEA3DedcA5984780Bafc599bD69ADd087D56")
	address := w3.A("0x06554dce7EC9000B9EA65b2C04a48256cC09c51F")
	
	var tokenBalance big.Int
	err := client.Call(eth.CallFunc(funcBalanceOf, contractAddress, address).Returns(&tokenBalance))
	if err != nil {
		log.Fatal(err)
	}
}

```

**Comment by lmittmann** [comment]
#16 was fixed in v0.9.1. Please upgrade to the latest version.
